config.py

```diff
@@ -233,6 +233,21 @@ class Config(object):
             ReturnCodes.TEMPORARY,
             (True, True)
         ),
+        'www.webmaker.org': (
+            'https://foundation.mozilla.org/en/artifacts/webmaker/',
+            ReturnCodes.PERMANENT,
+            (True, True)

```

Comment:
Make this tuple and the other two below `(False, True)` so only the query params are preserved

Reply:
In order to keep the ball rolling I committed a change suggestion for this

Reply:
Works for me!